haskell-src/exec/Chainweb/Server.hs

```diff
@@ -399,31 +399,15 @@ searchTxs logger pool req givenMbLim mbOffset (Just search) Nothing minheight ma
         resultLimit
       return $ maybe noHeader (addHeader . mkTxToken) mbCont $
         results <&> \(txwh,_) -> dbToApiTxSummary (txwhSummary txwh) (txwhContHistory txwh)
-searchTxs logger pool req givenMbLim mbOffset Nothing (Just pactid) minheight maxheight mbNext = do
+searchTxs logger pool req _givenMbLim _mbOffset Nothing (Just pactid) minheight maxheight mbNext = do
   liftIO $ logger Info $ fromString $ printf
     "Transaction search from %s: PactId string: %s" (show $ remoteHost req) (T.unpack pactid)
 
-  continuation <- mkContinuation readTxToken mbOffset mbNext
-
-  isBounded <- isBoundedStrategyM req
-
-  liftIO $ M.with pool $ \(c, throttling) -> do
-    let
-      scanLimit = ceiling $ 50000 * throttling
-      maxResultLimit = ceiling $ 250 * throttling
-      resultLimit = min maxResultLimit $ maybe 10 unLimit givenMbLim
-      strategy = if isBounded then Bounded scanLimit else Unbounded
+  let convert = undefined
 
-    PG.withTransactionLevel PG.RepeatableRead c $ do
-      (mbCont, results) <- performBoundedScan strategy
-        (runBeamPostgresDebug (logger Debug . T.pack) c)
-        (toTxSearchCursor . txwhSummary)
-        (txSearchSourcePactId pactid $ HeightRangeParams minheight maxheight)
-        noDecoration
-        continuation
-        resultLimit
-      return $ maybe noHeader (addHeader . mkTxToken) mbCont $
-        results <&> \(txwh,_) -> dbToApiTxSummary (txwhSummary txwh) (txwhContHistory txwh)
+  liftIO (M.with pool (queryTxsByPactId logger pactid . fst)) >>= \case
+    [] -> throw404 $ "No transactions found for pactid: " <> toS pactid
```

Comment:
I don't think we need to handle the empty results case specially here, returning an empty list of transactions should be fine.